db/migrate/20190130143545_change_test_run_errors_to_problems.rb

```diff
@@ -0,0 +1,6 @@
+class ChangeTestRunErrorsToProblems < ActiveRecord::Migration[5.2]
```

Comment:
Since I've asked for a few schema changes anyway, I'd prefer consolidating all the migrations into one file.

Reply:
Should I rewrite history for this? (otherwise, individual commits will be broken)